### PR TITLE
146 user about field

### PR DESCRIPTION
**Pull request by ervinj1998**
En esta rama agregamos un espacio donde podemos poner "acerca de."  y le damos unfion de retomar el estado anterioir.

### CODE REVIEW

**Comment by Hawaiideveloper** [comment]
@manzanarez83 please update all the green sections (additions) that @ervinj1998 added with your comments on what the code does.  It appears he had 7 commits(changes)

File: src/user/EditProfile.js [hunk 1]
```diff
@@ -26,15 +27,16 @@ export class EditProfile extends Component { //componente para habilitar el boto
...
+            if(data.error) {
+                this.setState({redirectToProfile: true });
+            } else {
+                this.setState({
+                    id: data._id, 
+                    name: data.name, 
+                    email: data.email,
+                    error: "",
+                    about: data.about //para retomar el estado anterior.
+            });
```
Comment (manzanarez83) [inline src/user/EditProfile.js:39]: I was able to see this change in the video when he added **about: data.about** in order to see the about field updated.

**Comment by Hawaiideveloper** [comment]
Screen Shot 2020-10-15 at 4 52 38 PM

**Comment by Hawaiideveloper** [comment]
looks good to me

**Pull request merged**